accounts/tests.py

```diff
@@ -1,3 +1,43 @@
 from django.test import TestCase
+from accounts.forms import UserRegisterForm
+from django.contrib.auth import get_user_model
+User = get_user_model()
 
-# Create your tests here.
+
+class SignUpTests(TestCase):
+
+    def test_signup_view(self):
+        response = self.client.post('/accounts/signup/')
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'registration/signup.html')
```

Comment:
Do you want to use this template on the POST?  I think you would use this template on the GET, when you get the form page to fill out and submit to create a new user object